Source/Core/Core/Core.cpp

```diff
@@ -534,6 +534,7 @@ static void EmuThread(std::unique_ptr<BootParameters> boot, WindowSystemInfo wsi
 
     PatchEngine::Shutdown();
     HLE::Clear();
+    PowerPC::debug_interface.Clear();
```

Comment:
Right after I hit the approve button, I realized I glossed over this.  What's this for?

Reply:
It clears the debug_interface after a full shutdown (i.e. breakpoints, patches, watches, memcheck) and it also releases the network logger and its file.

I have the feeling it's an artifact that wasn't migrated when switching to Qt, I might be wrong but I don't recall adding such line when I originally created the PR.

Reply:
Ah ok, looks like this might also resolve #9325 and the issue that was mentioned there?

Reply:
Sorry, my wording was poorly chosen. By "full shutdown" I meant the game/Wii shutdown not the actual game so it won't. 

After a closer look at the current code, I'll probably add a 2nd commit that removes the GUI code that takes care of the breakpoints/memchecks clearing as it's using Qt signals and Dolphin state rather than removing them on shutdown in core.

Reply:
Unnecessary GUI code removed in the 2nd commit.